src/js/common/schemaform/FormSaved.jsx

```diff
@@ -31,22 +32,33 @@ class FormSaved extends React.Component {
     }
   }
 
+  verifyUser = () => {
+    handleVerify(this.props.user.login.verifyUrl);
+  }
+
   render() {
     const { profile } = this.props.user;
     const lastSavedDate = this.props.lastSavedDate;
     const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
+    const verifiedAccountType = 3;// verified ID.me accounts are type 3
+    const notVerified = profile.accountType !== verifiedAccountType;
     const { success } = this.props.route.formConfig.savedFormMessages || {};
 
     return (
       <div>
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <strong>Your application has been saved!</strong><br/>
-            {!!lastSavedDate && <p>Last saved on {moment(lastSavedDate).format('M/D/YYYY [at] h:mma')}.</p>}
+            <strong>Your application has been saved.</strong><br/>
+            {!!lastSavedDate && <p>Last saved on {moment(lastSavedDate).format('M/D/YYYY [at] h:mm a')}</p>}
```

Comment:
Yes, in order to avoid a situation with "12:00 p.m..". Thanks for letting me know if this should be reverted.